recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/gles-user-module.bbappend

```diff
@@ -43,6 +45,8 @@ do_configure_prepend() {
     fi
 }
 
+do_compile[depends] += "linux-renesas:do_shared_workdir"
```

Comment:
why?
you have DEPENDS on virtual/kernel, isn't it enough?
if, for some reason, you still need this dependency, then put dependency on do_build (which is the last task) rather then do_shared_workdir

Reply:
pls put "https://stackoverflow.com/questions/34793697/how-to-write-a-bitbake-driver-recipe-which-requires-kernel-source-header-files" link to describe the problem

Reply:
Done